src/DeviceInput/InputDevices/nativeDeviceInputSystem.ts

```diff
@@ -22,18 +51,11 @@ export class NativeDeviceInputSystem implements IDeviceInputSystem {
             this.onDeviceDisconnected(deviceType, deviceSlot);
         };
 
-        this._nativeInput.onInputChanged = (deviceType, deviceSlot, inputIndex, previousState, currentState, eventData) => {
-            const idx = (inputIndex === PointerInput.Horizontal || inputIndex === PointerInput.Vertical || inputIndex === PointerInput.DeltaHorizontal || inputIndex === PointerInput.DeltaVertical) ? PointerInput.Move : inputIndex;
-            const evt = DeviceEventFactory.CreateDeviceEvent(deviceType, deviceSlot, inputIndex, currentState, this);
-
-            let deviceEvent = evt as IDeviceEvent;
-            deviceEvent.deviceType = deviceType;
-            deviceEvent.deviceSlot = deviceSlot;
-            deviceEvent.inputIndex = idx;
-            deviceEvent.previousState = previousState;
-            deviceEvent.currentState = currentState;
+        this._nativeInput.onInputChanged = (deviceType, deviceSlot, inputIndex, previousState, currentState) => {

```

Comment:
I guess `previousState` is no longer used? Should it be removed from here and from BN?

Reply:
That would make sense.  Removed from JS.  I'll update my BN PR to reflect this.